handlers.py

```diff
@@ -0,0 +1,146 @@
+from telegram import Location, Update
+from telegram.ext import CallbackContext, CommandHandler, MessageHandler, filters
+from constants import RESTAURANT
+from decorators import check_start, user_started
+from logger import logger
+from wolt_exception import WoltException
+from wolt_services import is_restaurant_open
+
+restaurant_index: int | None = None
+restaurant_is_open: bool | str = False
+lat: float | None = None
+lon: float | None = None
```

Comment:
Ok this is a big one:
let's assume this piece of code is running as a backend on your favourite cloud provider and serving hundreds of users at a time. What will happen when one user checks at the same time for restaurant A and a different user checks for restaurant B?

Hint:
incosistent data.

2nd hint:
use dictionaries.

Reply:
I'll do you one better. Even if I do a dictonary where every key is a user ID which holds the lat, lon and the restaurant he chose, I will have memory problems and the application will get quite slow when there will be thousands of users. also if I want to let's say have multiple instances of the bot running I will still have problems of inconsistent data.

 For the sport I will add a redis instance using docker where  I will hold the data of the users. and also add a dockerfile for this app.